cloud/pkg/cloudhub/servers/httpserver/signcerts.go

```diff
@@ -61,7 +61,7 @@ func getIps(advertiseAddress []string) (Ips []net.IP) {
 
 // GenerateToken will create a token consisting of caHash and jwt Token and save it to secret
 func GenerateToken() error {
-	expiresAt := time.Now().Add(time.Hour * 24).Unix()
+	expiresAt := time.Now().Add(time.Hour * hubconfig.Config.CloudHub.TokenRefreshTime * 2).Unix()
```

Comment:
plz add comment above to show why time need to multiply 2.
And I recommend this value close to and slightly greater than  1.